Entities/Items/Crate/Crate.as

```diff
@@ -139,6 +143,25 @@ void onTick(CBlob@ this)
 	}
 }
 
+void PickupOverlap(CBlob@ this)
+{
+	if (getNet().isServer())
+	{
+		Vec2f tl, br;
+		this.getShape().getBoundingRect(tl, br);
+		CBlob@[] blobs;
+		this.getMap().getBlobsInBox(tl, br, @blobs);
+		for (uint i = 0; i < blobs.length; i++)
+		{
+			CBlob@ blob = blobs[i];
+			if (!blob.isAttached() && blob.isOnGround() && blob.hasTag("material") && blob.getName() != "mat_arrows")
+			{
+				this.server_PutInInventory(blob);
+			}
+		}
+	}
+}
```

Comment:
Duplicate from `Storage.as`, this function should be merged into a common script for `Crate.as` and `Storage.as`.

Reply:
Alrighty, wasn't sure if I should do that or not. Maybe I should make it a separate script instead? (not a common, something like AutoPickupMats.as)